pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -317,14 +317,14 @@ func (f *framework) RunPreFilterUpdaterAddPod(pc *PluginContext, podToSchedule *
 	return nil
 }
 
-// RunPreFilterUpdaterRemovePod calls the RemovePod interface for the set of configured
+// RunPreFilterExtensionRemovePod calls the RemovePod interface for the set of configured
 // PreFilter plugins. It returns directly if any of the plugins return any
 // status other than Success.
-func (f *framework) RunPreFilterUpdaterRemovePod(pc *PluginContext, podToSchedule *v1.Pod,
+func (f *framework) RunPreFilterExtensionRemovePod(pc *PluginContext, podToSchedule *v1.Pod,
 	podToRemove *v1.Pod, nodeInfo *schedulernodeinfo.NodeInfo) *Status {
 	for _, pl := range f.preFilterPlugins {
-		if updater := pl.Updater(); updater != nil {
-			status := updater.RemovePod(pc, podToSchedule, podToRemove, nodeInfo)
+		if extension := pl.Extensions(); extension != nil {
```

Comment:
`extensions` plural instead?

Reply:
really just looking for consistency

Reply:
Sorry for my careless,  I have changed extension to extensions.